scripts/apps/search/components/PhotoDeskInfo.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {createMarkUp} from '../helpers';
+
+/**
+ * PhotoDesk Info - renders item metadata
+ */
+export function PhotoDeskInfo(props) {
```

Comment:
you can use jsx in other files too, not just here ;)

Reply:
😑